lib/Elastica/Client.php

```diff
@@ -88,6 +93,21 @@ public function __construct(array $config = [], $callback = null, LoggerInterfac
     }
 
     /**
+     * Get current version
+     *
+     * @return string
+     */
+    public function getVersion()
+    {
+        if ($this->_version) {
+            return $this->_version;
```

Comment:
It's possible. Imagine two _repositories_: **A**, **B** both are using ES, both should be B/C with 1.7 and 2.0.